hasher-matcher-actioner/hma-lite/hmalite/matcher.py

```diff
@@ -0,0 +1,24 @@
+from flask import Blueprint, request, jsonify
+
+matcher_api = Blueprint('matcher_api', __name__)
+
+@matcher_api.route('/query', methods=['GET', 'POST'])
+def matcher_query():
+    if request.method == 'POST':
+        hashes = request.json['hashes']
+        matches = [ hash for hash in hashes if _mock_is_index_match(hash)]
+        return jsonify({
+            "matched_hashes": matches,
+            "unmatched_hashes": [hash for hash in hashes if hash not in matches]
+        })
+    else:
+        hash = request.args.get('hash')
+        if _mock_is_index_match(hash):
+            return ('', 200)
```

Comment:
Eventually there should be something returned in the body, I think, not just empty string (maybe more metadata about what the hash matches?) Just thinking out loud.